CodeReview/src/commands/CCodeReviewComment.cpp

```diff
@@ -49,12 +54,24 @@ bool CCodeReviewComment::canInterpret(Visualization::Item*, Visualization::Item*
 	return false;
 }
 
-Interaction::CommandResult* CCodeReviewComment::execute(Visualization::Item*, Visualization::Item*,
-				const QStringList&, const std::unique_ptr<Visualization::Cursor>& cursor)
+Interaction::CommandResult* CCodeReviewComment::execute(Visualization::Item* source, Visualization::Item*,
+				const QStringList&, const std::unique_ptr<Visualization::Cursor>&)
 {
-	auto cursorOwner = cursor->owner();
-	auto overlay = new CodeReviewCommentOverlay{cursorOwner};
-	cursorOwner->addOverlay(overlay, "CodeReviewComment");
+	for (auto manager : Model::AllTreeManagers::instance().loadedManagers())
+		if (manager->nodeIdMap().contains(source->node()))
+		{
+			auto id = manager->nodeIdMap().id(source->node()).toString();
+			auto commentedNode = CodeReviewManager::instance().commentedNode(id);
+			commentedNode->commentNodes()->append(new Comments::CommentNode{"comment here"});
+
+			// only create highlight if not already existent
```

Comment:
This way you can have sort of a "thread" for a node. It displays a list with the comments which gives the impression that multiple authors are discussing an issue. Of course you can just edit the first comment and don't add anything new. So it's not needed and more of a test :)